modules/peers.js

```diff
@@ -336,6 +366,11 @@ __private.dbLoad = function(cb) {
 			async.each(
 				rows,
 				(peer, eachCb) => {
+					// Ban peer if it is presented in the array of black listed peers
+					if (__private.isBlacklisted(peer.ip)) {
+						peer.state = Peer.STATE.BANNED;
+					}
+
 					peer = library.logic.peers.create(peer);
 					if (library.logic.peers.upsert(peer, true) !== true) {
 						return setImmediate(eachCb);
```

Comment:
Maybe we should change `peer.state > 0` below (on line 378 and also line 838) to `peer.state !== Peer.STATE.BANNED` to be more clear/explicit.

Reply:
Done